### PR TITLE
[Python] Challenge 7 (Unreviewed)

### PR DESCRIPTION
**Pull request by sarcodian**

### CODE REVIEW

File: challenge_7/python/sarcodian/src/challenge_7.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+def missing_int(L):
+
+    sum_n = 0
+    sum_n_less_1 = 0
+
+    for i in range(len(L)+1):
+        sum_n += i
+    
+    for i in L:
+        sum_n_less_1 += i
```
Comment (ningyuansg) [inline challenge_7/python/sarcodian/src/challenge_7.py]: Try the more pythonic and readable `sum(L)`.

File: challenge_7/python/sarcodian/src/challenge_7.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+def missing_int(L):
+
+    sum_n = 0
+    sum_n_less_1 = 0
+
+    for i in range(len(L)+1):
+        sum_n += i
```
Comment (ningyuansg) [inline challenge_7/python/sarcodian/src/challenge_7.py]: Try the more pythonic `sum([i for i in range(len(L)+1)])`

Comment (sarcodian) [inline challenge_7/python/sarcodian/src/challenge_7.py]: Thanks, can you provide a link or something that provides a deeper explanation of <i for i in range(len(L)+1)> syntax. I have seen it a lot in other code as well, but I don't get the logic behind how it works so I am wary of just using it. Or even if you provide the name for what that type of syntax is called I can google it.

File: challenge_7/python/sarcodian/src/challenge_7.py (hunk 1 above)
Comment (sarcodian) [inline challenge_7/python/sarcodian/src/challenge_7.py]: Haha, thanks, didn't know it existed :)

File: challenge_7/python/sarcodian/src/challenge_7.py (hunk 2 above)
Comment (ningyuansg) [inline challenge_7/python/sarcodian/src/challenge_7.py]: As sent to you in slack, here's a great video on the topic (and also for the reference of others)
https://www.youtube.com/watch?v=ZoWgzG_r2qo

**Pull request merged**